src/builders/indexable-link-builder.php

```diff
@@ -178,6 +178,36 @@ protected function gather_links( $content ) {
 	 * @return string[] An array of urls.
 	 */
 	protected function gather_images( $content ) {
+		$images = $this->gather_images_from_content( $content );
+		\array_merge( $images, $this->get_featured_image() );
+
+		return $images;
+	}
+
+	/**
+	 * Gathers all images from content.
+	 *
+	 * @return string[] An array of urls.
+	 */
+	protected function get_featured_image() {
```

Comment:
Feels a bit weird to have this return an array, I'd rather return either a string or `null` and then optionally add it above.